api-contracts/openapi/components/schemas/metadata.yaml

```diff
@@ -105,9 +105,8 @@ APIResourceMeta:
       type: string
       description: "the id of this resource, in UUID format"
       example: bb214807-246e-43a5-a25d-41761d1cff9e
-      minLength: 36
+      minLength: 0
```

Comment:
I'm not sure how we want to handle this, I chose bigint for labels but still want to use tables...